<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(C:/Users/herme/Desktop/BlackJack UI/assets/images/download.jpeg)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="p_card_1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="p_card_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="p_card_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="p_card_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="p_card_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="p_card_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="d_card_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid yellow;
border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: Georgia, serif;
color: white;
background-image: url(&quot;&quot;);</string>
    </property>
    <property name="text">
     <string>Krupiye</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: Georgia, serif;
color: white;
background-image: url(&quot;&quot;);</string>
    </property>
    <property name="text">
     <string>Oyuncu</string>
    </property>
   </widget>
   <widget class="QLabel" name="playerPoints">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;background-image: url(&quot;&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dealerPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
opacity: 0%;background-image: url(&quot;&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="moneyLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="betLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
font-family: Georgia, serif;
color: white;background-image: url(&quot;&quot;);</string>
    </property>
    <property name="text">
     <string>Bahis:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>260</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:/Users/herme/Desktop/BlackJack UI/assets/images/black.jpg);
border: 2px outset #5A002C;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="hitButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008000;
border-radius: 30px;
border: 2px solid grey;
border-style: outset;
background-image: url(&quot;&quot;);
color: white;
font-size: 14px</string>
     </property>
     <property name="text">
      <string>Kart Çek</string>
     </property>
    </widget>
    <widget class="QPushButton" name="standButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #880808;
border-radius: 30px;
border: 2px solid grey;
border-style: outset;
background-image: url(&quot;&quot;);
color: white;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Kal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="standButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;
border-radius: 30px;
border: 2px solid grey;
border-style: outset;
background-image: url(&quot;&quot;);
color: white;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Teslim Ol</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="chip50Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;

border-style: outset;
border-image: url(C:/Users/herme/Desktop/BlackJack UI/assets/images/chip50.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chip10Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;

border-style: outset;
border-image: url(C:/Users/herme/Desktop/BlackJack UI/assets/images/chip10.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chip100Button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
border-image: url(C:/Users/herme/Desktop/BlackJack UI/assets/images/chip100.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="betButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Bahsi Onayla</string>
    </property>
   </widget>
   <widget class="QLabel" name="betLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px groove grey;
background-image: url(&quot;&quot;);
background-color: green;


</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
</string>
     </property>
     <property name="text">
      <string>Devam</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 0px solid white;
font-size: 20px;
font-family: Georgia, serif;</string>
     </property>
     <property name="text">
      <string>asdasdasdafasf</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>p_card_1</zorder>
   <zorder>d_card_1</zorder>
   <zorder>p_card_2</zorder>
   <zorder>p_card_3</zorder>
   <zorder>p_card_4</zorder>
   <zorder>p_card_5</zorder>
   <zorder>p_card_6</zorder>
   <zorder>d_card_2</zorder>
   <zorder>d_card_3</zorder>
   <zorder>d_card_4</zorder>
   <zorder>d_card_5</zorder>
   <zorder>d_card_6</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>playerPoints</zorder>
   <zorder>dealerPoints</zorder>
   <zorder>moneyLabel</zorder>
   <zorder>betLabel</zorder>
   <zorder>label_3</zorder>
   <zorder>chip50Button</zorder>
   <zorder>chip10Button</zorder>
   <zorder>chip100Button</zorder>
   <zorder>betButton</zorder>
   <zorder>betLabel_2</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
